### PR TITLE
LG-2960: set region on s3 client

### PR DESCRIPTION
**Pull request by solipet**

### CODE REVIEW

File: app/controllers/service_providers_controller.rb [hunk 1]
```diff
@@ -166,7 +166,7 @@ def using_s3?
   end
 
   def s3
-    @s3 ||= Aws::S3::Client.new
+    @s3 ||= Aws::S3::Client.new region: Figaro.env.aws_region
```
Comment (zachmargolis) [inline app/controllers/service_providers_controller.rb:169]: style: I'd use parens here please but no big deal

File: app/services/service_provider_logo_updater.rb [hunk 2]
```diff
@@ -105,6 +105,8 @@ def dashboard_sp_url
     else
       'http://localhost:3001/api/service_providers'
     end
+  rescue LoginGov::Hostdata::MissingConfigError
```
Comment (zachmargolis) [inline app/services/service_provider_logo_updater.rb:108]: Looking at the source of the gem....why is this rescue needed? the `in_datacenter?` guard should cover this? Are we stubbing part of the gem or something in specs? If so, I'd look into how we can maybe not stub it, or maybe use `Hostdata.reset!` to clean up any garbage state?

Comment (solipet) [reply app/services/service_provider_logo_updater.rb]: yup, too much stubbing

**Comment by solipet** [comment]
seriously? diff coverage?!

**Comment by zachmargolis** [comment]
(Used admin privs to merge and override the code coverage diff errors, they were not significant)

**Pull request merged**